test/cqlpy/test_describe.py

```diff
@@ -183,19 +183,31 @@ def test_desc_scylla_keyspace(scylla_only, cql, random_seed):
 
 # Test that `DESC TABLE {tbl}` contains appropriate create statement for table
 # This test compares the content of `system_schema.tables` and `system_schema.columns` tables.
-def test_desc_table(cql, test_keyspace, random_seed, has_tablets):
+@pytest.mark.parametrize("with_tablet_hints", [False, True])
+def test_desc_table(cql, test_keyspace, random_seed, has_tablets, with_tablet_hints:bool):
```

Comment:
I am not thrilled with the dtest-like tests that test everything in one function. 
When you add a new feature ("tablet hints"), I think you should usuallly leave the existing tests alone and create a new test specifically for the new feature. The new test doesn't need to copy _everything_ that the previous test had - it can check just the tablet hints and nothing else.

Reply:
can do

Reply:
Fixed in v3